### DISCUSSION TITLE
Hekili suitable for leveling?

### DISCUSSION THREAD

**Discussion by fredhsv96** [Hekili/hekili#1324] [Q&A]
Hey, 

After a long break I am leveling up a new character again.

To deal with the Rota/Priolist during the leveling, I have installed Hekili.

Unfortunately I get skills that I don't have yet.

Therefore the question is Hekili exclusively designed for max level or do I have to change something in the settings?

Thanks

**Comment by Hekili** [comment]
Every time you log in with the addon, it will say:

> Hekili is designed for current content.
> Use below level 50 at your own risk.

For some specs, it's fine.  For others, it's not.  But it's as-is, no warranty.

**Comment by fredhsv96** [comment]
ok thanks for the answer. i actually missed the message, sorry. i'm just surprised that other rotation helpers can do this, so that they only show skills that you actually already have or have skilled.

**Comment by Hekili** [comment]
If other developers want to commit time to optimizing for content you won't spend more than a few hours in, they're welcome to do so!  Use their addons if they do a good job of it.

**Comment by fredhsv96** [comment]
Sorry for asking...

**Comment by Hekili** [comment]
No need to be sorry.  I truly mean it, though.  This addon has specific design goals, largely centered around providing recommendations consistent with sim profiles.  That has advantages and disadvantages.  If there are other addons that are built around the changes that occur while leveling and provide good guidance throughout the leveling process, there's no reason not to use them when you're leveling.
